packages/leafygreen-provider/src/LeafyGreenProvider.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import UsingKeyboardProvider from './UsingKeyboardProvider';
+
+interface LeafyGreenProviderProps {
+  children?: React.ReactNode;
+}
+
+function LeafyGreenProvider({ children }: LeafyGreenProviderProps) {
+  return <UsingKeyboardProvider>{children}</UsingKeyboardProvider>;
```

Comment:
Yes, say we add some random provider. It would look like:
```js
return (
  <UsingKeyboardProvider>
    <NewProvider>
      {children}
    </NewProvider>
  </UsingKeyboardProvider>
```

As far as common provider design patterns, I don't think there's another great way to do this. We could have one combined state provider, but that would probably cause a few more renders than would be ideal, and doesn't let us isolate the functionality as much.